overlord/state/notices_test.go

```diff
@@ -679,6 +679,93 @@ func (s *noticesSuite) TestValidateNotice(c *C) {
 	c.Check(id, Equals, "")
 }
 
+func (s *noticesSuite) TestAvoidTwoNoticesWithSameDateTime(c *C) {
```

Comment:
It seems to me this test is testing too much. All we need to test is that two notices with the same (fake) timeNow end up with notice2.lastOccurred > notice1.lastOccurred, right?

Reply:
It's better to test three, to avoid the error that could result in using `Equal()` instead of `!After()`.

Reply:
Done. Now it uses `noticeToMap()`.